src/utils/geocoder.ts

```diff
@@ -53,15 +53,15 @@ export const geocoderApi = {
       const geojson = await response.json()
       features = handleNominatimResponse(geojson)
     }
-    catch (e) {
+    catch (e: any) {
       if (e.name !== 'AbortError')
         console.error(`Failed to forwardGeocode with error: ${e}`)
     }
     return {
       features,
     }
   },
-  reverseGeocode: async (config: { query: string }) => {
+  reverseGeocode: async (config: { query: any }) => {
```

Comment:
any type should be precised ?

Reply:
`maplibre-gl-geocoder` is not typescripted and query is defined in the doc to be a "Object"